Week1/homework/app.js

```diff
@@ -1,11 +1,161 @@
+    'the_adventures_of_huckleberry_finn',
   ];
-
   // Replace with your own code
-  console.log(bookTitles);
-}
+  const infoForEachBook = {
+    harry_potter_and_the_chamber_of_secrets: {
+      title: 'Harry Potter and the Chamber of Secrets',
+      author: ' J. K. Rowling',
+      genre: 'Fantasy',
+      language: 'English',
+      country: 'United Kingdom',
+    },
+    the_art_of_war: {
+      title: 'The Art of War',
+      author: 'Sun Tzu',
+      genre: 'Historical fiction',
+      language: 'Chinese',
+      country: 'China',
+    },
+    the_rise_and_fall_of_the_great_powers: {
+      title: 'The Rise and Fall of the Great Powers',
+      author: 'Paul Kennedy',
+      genre: 'Economics, History',
+      language: 'English',
+      country: 'United States',
+    },
+    the_brothers_karamazov: {
+      title: 'The Brothers Kazamazov',
+      author: ' Fyodor Dostoevsky',
+      genre: 'Philosophical novel',
+      language: 'Russian',
+      country: 'Russia',
+    },
+    in_search_of_lost_time: {
+      title: 'In Search of Lost Time',
+      author: ' Marcel Proust',
+      genre: 'Modernist',
+      language: 'French',
+      country: 'France',
+    },
+    great_expectations: {
+      title: 'Great Expectations',
+      author: ' Charles Dickens',
+      genre: 'Novel',
+      language: 'English',
+      country: 'United Kingdom',
+    },
+    war_and_peace: {
+      title: 'War and Peace',
+      author: ' Leo Tolstoy',
+      genre: 'Historical novel',
+      language: 'Russian',
+      country: 'Russia',
+    },
+    the_great_gatsby: {
+      title: 'The Great Gatsby',
+      author: 'F. Scott Fitzgerald',
+      genre: 'Novel',
+      language: 'English',
+      country: 'United States',
+    },
+    to_kill_a_mockingbird: {
+      title: 'To Kill a Mockingbird',
+      author: 'Harper Lee',
+      genre: ' Southern Gothic, Bildungsroman',
+      language: 'English',
+      country: 'United States',
+    },
+    the_adventures_of_huckleberry_finn: {
+      title: 'The Adventures of Huckleberry Finn',
+      author: 'Mark Twain',
+      genre: 'Picaresque novel',
+      language: 'English',
+      country: 'United States',
+    },
+  };
+
+  
+  const bookCovers = {    
+    harry_potter_and_the_chamber_of_secrets: './img/harry_potter_and_the_chamber_of_secrets.jpg',
+    the_art_of_war: ' ./img/the_art_of_war.jpg',
+    the_rise_and_fall_of_the_great_powers: ' ./img/the_rise_and_fall_of_the_great_powers.jpg',
+    the_brothers_karamazov: ' ./img/the_brothers_karamazov.jpg',
+    in_search_of_lost_time: './img/in_search_of_lost_time.jpg',
+    great_expectations: ' ./img/great_expectations.jpg',
+    war_and_peace: ' ./img/war_and_peace.jpg',
+    the_great_gatsby: ' ./img/the_great_gatsby.jpg',
+    to_kill_a_mockingbird: ' ./img/to_kill_a_mockingbird.jpg',
+    the_adventures_of_huckleberry_finn: ' ./img/the_adventures_of_huckleberry_finn.jpg',
+  };
+
+
+  const newDiv = document.createElement('div');
```

Comment:
How "new" is this `<div>`? New div isn't the best name for this element and variable. Maybe you could use a semantic element and give it a more descriptive name. Same goes for `newH1`, `newUl`, `newImg` etc.

Reply:
I changed all the names starting with “new”. My intention was to show that I created new elements but it was not a good idea.